_rules/form-control-label-descriptive-cc0f0a.md

```diff
@@ -92,17 +102,6 @@ Label is [visible][], but not included in accessibility tree
 <input aria-labelledby="label_fname" type="text" name="fname" />
 ```
 
-#### Passed Example 5
```

Comment:
This is the main reason to change this rule. This test case does not pass SC 2.4.6.

Reply:
How would this fail SC 2.4.6? The label is descriptive. I think this fail SC 3.3.2 (depending on your definition of "provided".